target/linux/kirkwood/base-files/etc/board.d/01_leds

```diff
@@ -17,6 +17,9 @@ linksys,e4200-v2|\
 linksys,ea4500)
 	ucidef_set_led_default "pulse" "pulse" "viper:white:pulse" "1"
 	;;
+globalscale,sheevaplug)
+    ucidef_set_led_timer "health" "health" "sheevaplug:blue:health" "200" "800"
+    ;;
```

Comment:
And please use the correct indentation based on what the rest of the file is like.

Reply:
Dont know how that happend. Fixed,